### PR TITLE
Fix upgrade documentation and other...

### PR DESCRIPTION
**Pull request by sbrunner**
replace: #2700,  #2699, #2698, #2695

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -14,6 +14,14 @@ If you want to keep the `CGXP` interface please read the following
 documentation and then move directly to step 4:
 https://camptocamp.github.io/c2cgeoportal/2.0/integrator/legacy_cgxp.html
 
+After you upgrade the application you can use the following scripts to help you on upgrade
+your data fron CGXP to ngeo:
+* `.build/venv/bin/themev1tov2` to migrate the them
```
Comment (pgiraud) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: them -> themes

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 2]
```diff
@@ -14,6 +14,14 @@ If you want to keep the `CGXP` interface please read the following
...
+* `.build/venv/bin/l10nv1tov2` to migrate the localisation
```
Comment (pgiraud) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: localisation -> language strings

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 3]
```diff
@@ -14,6 +14,14 @@ If you want to keep the `CGXP` interface please read the following
...
+Use the `--help` argument for more information.
+
+You should also add s Layer group named 'background' with the background layers.
```
Comment (pgiraud) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: s -> a

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 4]
```diff
@@ -251,24 +254,23 @@ replace your sencha touch interface by an ngeo interface.
...
 
-8. In the `{{package}}/templates/(index|edit|routing).html` files do the following change:
+7. In the `{{package}}/templates/(index|edit|routing).html` files do the following change:
 
       <input id="password" name="password" type="password" autocomplete="on"/>
     + <input id="oldPassword" name="oldPassword" type="password" />
       <input id="newPassword" name="newPassword" type="password" />
 
-9. On the `{{package}}/models.py` removes the following lines:
+8. On the `{{package}}/models.py` removes the following lines:
```
Comment (pgiraud) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: On -> In
removes -> remove

**Comment by sbrunner** [comment]
Thanks @pgiraud :-)

**Pull request closed without merging**